src/api/data_types/chunking/mobile_app.rs

```diff
@@ -9,9 +9,24 @@ pub struct ChunkedMobileAppRequest<'a> {
     pub checksum: Digest,
     pub chunks: &'a [Digest],
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub git_sha: Option<&'a str>,
-    #[serde(skip_serializing_if = "Option::is_none")]
     pub build_configuration: Option<&'a str>,
+    // VCS fields
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub head_sha: Option<&'a str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub base_sha: Option<&'a str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub provider: Option<&'a str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub head_repo_name: Option<&'a str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub base_repo_name: Option<&'a str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub head_ref: Option<&'a str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub base_ref: Option<&'a str>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub pr_number: Option<i32>,
```

Comment:
As PR's can only have positive numbers, I would use an unsigned integer (`u32`) here, rather than a signed integer

```suggestion
    pub pr_number: Option<u32>,
```